test/integration/examples/apiserver_test.go

```diff
@@ -226,18 +231,21 @@ func TestAPIServiceWaitOnStart(t *testing.T) {
 }
 
 func TestAggregatedAPIServer(t *testing.T) {
-	t.Run("WithoutWardleFeatureGateAtV1.2", func(t *testing.T) {
```

Comment:
this test is missing.

Reply:
This is intentional. This is because if we use 1.2 and kube 1.31, we cannot test N-1 emu version, since setting kube emu version to 1.30 is not allowed for this release, that would be fixed in the next release.

Reply:
Added it back to test the default options.